src/lsst/cmservice/web_app/templates/group_details.html

```diff
@@ -27,10 +27,146 @@
   </ol>
 </nav>
 <div class="w-full">
-    <div class="overflow-hidden px-4 py-5 sm:p-6 relative w-1/4" xmlns="http://www.w3.org/1999/html">
+    <div class="overflow-hidden px-4 py-5 sm:p-6 relative w-1/2" xmlns="http://www.w3.org/1999/html">
```

Comment:
And here again? If this is required maybe add it to the html tag in base?

Reply:
Not required. removed!